src/community/design-system-working-group/index.md.njk

```diff
@@ -20,8 +20,14 @@ By including representatives from a mixture of disciplines and departments, the
 
 It means that decisions made are fair and unbiased, and that the Design System reflects the experiences of the whole of government not just one department. 
 
-## How to join the working group
+## Join the working group
 
-The GOV.UK Design System team is looking into the best way of helping people to join the working group.
+We’re looking for new members to help make the working group more diverse and better represent the wider public sector.
 
-In the meantime if you are interested in joining the working group email the Design System team at govuk-design-system-support@digital.cabinet-office.gov.uk
+If you think you can expand our range of voices across ethnicity, age, gender, experience and discipline, we want to hear from you.
+
+You’ll help promote collaboration across government and the Digital Transformation Strategy. Although our members volunteer their time with us, many of them do so as part of their community or corporate objectives.
```

Comment:
@calvin-lau-gds can we find a link to the Digital Transformation Strategy?

Reply:
We could even frame it around solving common user problems across government.

Reply:
Probably link here?
https://www.gov.uk/government/publications/government-transformation-strategy-2017-to-2020

I've added a suggested edit to this gDoc:
https://docs.google.com/document/d/1jHqqb4IyTgv35oWgZynQSSyKhAOxW25snqWr_hdvnZw/edit?usp=sharing